lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -233,9 +233,16 @@ click_secondary_menu:
   - selector:
       xpath: //a[@data-component='pf-nav-expandable' and text()='<primary_menu>']
     op: click
-  - selector:
+  - selector: &secondary_menu
       xpath: //a[@class='pf-c-nav__link' and text()='<secondary_menu>']
     op: click
+check_secondary_menu:
```

Comment:
Looks like it's not used, so can we just remove it?

Reply:
https://github.com/openshift/cucushift/blob/d736063a90bb9100e221f5567f2df12c2f90c910/web/admin-console/user-management.feature#L43